src/main/kotlin/com/exactpro/th2/codec/AbstractSyncCodec.kt

```diff
@@ -53,6 +54,7 @@ abstract class AbstractSyncCodec(
         var protoResult: MessageGroupBatch? = null
 
         try {
+            logger.debug { "codec receive ${TextFormat.shortDebugString(message)}" }
```

Comment:
Why do we need that? We can see received messages in mq router logs

Reply:
true